app/views/qsets/_modal_edit_qset.html.erb

```diff
@@ -29,6 +29,21 @@
               <%= permissions.label :questions_visible_to_unauth_user, 'Questions are visible to unauthenticated users' %>
               <%= permissions.check_box :questions_visible_to_unauth_user, {}, 'true', 'false' %>
             </div>
+            <div class="form-group">
+              <%= permissions.label :qset_type, 'Qset type:' %>
```

Comment:
After our discussion today I took a look at the documentation and tried to simplify this. It turns out Rails does all this `==` magic for you... it can be simplified down to:

```
            <div class="form-group">
              <%= permissions.label :qset_type, 'Qset type:' %>
              <%= permissions.label :qset_type_questions, 'Questions' %>
              <%= permissions.radio_button :qset_type, 'questions' %>
              <%= permissions.label :qset_type_subsets, 'Subsets' %>
              <%= permissions.radio_button :qset_type, 'subsets' %>
            </div>
```

As for using the specific labels, instead of inline text, see http://guides.rubyonrails.org/form_helpers.html#helpers-for-generating-form-elements (sorry for the not-very-specific link, the rails documentation isn't that great -- see the note at the end of section 1.3.2). tl;dr is that screen readers can associate text to options that way, and also clicking on the label will select that radio button.